WordPress/src/main/java/org/wordpress/android/ui/uploads/PostMediaHandler.kt

```diff
@@ -15,6 +15,7 @@ class PostMediaHandler
             val mediaForPost = mediaStore.getMediaForPost(post)
             mediaForPost.filter { it.postId == 0L }.forEach { media ->
                 media.postId = post.remotePostId
+                media.fieldsToUpdate = arrayOf("parent_id")
```

Comment:
This is being added within the `updateMediaWithoutPostId` function, which is called when unattached media is uploaded to a post (background can be found in https://github.com/wordpress-mobile/WordPress-Android/pull/13269). As such, it's hoped that only updating the media item's `parent_id` is necessary here and any updates to other fields would be unnecessary.